### DISCUSSION TITLE
GitHub Packages: Error response from daemon: denied

### DISCUSSION THREAD

**Discussion by ijpatricio** [serversideup/docker-php#256] [Q&A] [answered]
Hey Jay!

Trying the goodies!! It seems GH packages is not working?

```
docker pull ghcr.io/serversideup/php:beta-unit

Error response from daemon: Head "https://ghcr.io/v2/serversideup/php/manifests/beta-unit": denied: denied
```

All good for Docker Hub, it seems.

<img width="856" alt="image" src="https://github.com/serversideup/docker-php/assets/26031459/7c6c37c4-fb45-49da-8893-37fb47cbc8cd">

**Comment by jaydrogers** [comment]
Whoops, somehow this got buried in my inbox.

Can you try again? I think I fixed this a few weeks back in my tagging. I don't think the tag existed before, now it should.

I just tested this on my end and it worked 👍

**Comment by ijpatricio** [comment]
No problemo!

Unfortunately, same result, GHCR not working. Docker Hub yes!

![image](https://github.com/serversideup/docker-php/assets/26031459/7c346e8e-7d01-4511-8243-b6dabb97e930)

**Comment by ijpatricio** [comment, marked as answer]
Don't know why locally was failing to me. Maybe because I have some credentials logged in to GHCR??

My bad!

Working, can close!

![image](https://github.com/serversideup/docker-php/assets/26031459/78d4460c-b18a-4a0d-b1c7-d14ec87af782)
